aten/src/ATen/native/TensorShape.cpp

```diff
@@ -798,8 +799,18 @@ Tensor diag_embed(const Tensor& self, int64_t offset, int64_t dim1_, int64_t dim
   return result;
 }
 
-Tensor expand(const Tensor& self, IntArrayRef size, bool /*unused*/) {
-  TORCH_CHECK(size.size() >= (size_t)self.dim(),
+IntArrayRef AssertConcreteSizes(c10::ArrayRef<c10::SymbolicOrConcreteInt> ar) {
```

Comment:
Can this be part of the automatic conversion c10::ArrayRef<c10::SymbolicOrConcreteInt> -> IntArrayRef ?

Reply:
I think so. Though I haven't tried it yet.
I'm still thinking back to @zou3519 's point that it might not be the best UX even though practically it might gives us a better ergonomics

Reply:
If the conversion can fail we shouldn't allow it to be implicit. Have to ask for it explicitly.